libraries/script-engine/src/ScriptEngine.cpp

```diff
@@ -1550,6 +1550,13 @@ void ScriptEngine::unloadAllEntityScripts() {
 #endif // DEBUG_ENGINE_STATE
 }
 
+EntityScriptDetails ScriptEngine::getEntityScriptDetails(const EntityItemID& entityID) const {
```

Comment:
This is kind of a duplicate. I wrote a version that looks like:
`bool ScriptEngine::getEntityScriptDetails(const EntityItemID& entityID, EntityScriptDetails &details) const`